Inferences/Factoring.cpp

```diff
@@ -105,7 +105,6 @@ class Factoring::ResultsFn
   : _cl(cl), _cLen(cl->length()), _afterCheck(afterCheck), _ord(ord) {}
   Clause* operator() (pair<Literal*,RobSubstitution*> arg)
   {
-    if (_cl->hasAnswerLiteral() && !arg.second->apply(_cl->getAnswerLiteral(), 0)->computableOrVar()) return 0;
```

Comment:
Thank you, @hzzv , for immediately jumping to action. However, should if-condition in Factoring really disappear completely?

Reply:
Actually it can. We now have the `InvalidAnswerLiteralRemoval` inference which removes clauses with uncomptuable arguments in answer literals. This even further decreases the side effects on the main flow when synthesis isn't on.

Reply:
Perfect!